src/job-execution/WebJobExecutor.ts

```diff
@@ -169,19 +169,27 @@ export class WebJobExecutor extends BaseJobExecutor {
               ? res.result.log.map((logLine: any) => logLine.line).join('\n')
               : res.result.log
 
-          const resObj = res
-
-          if (this.serverType === ServerType.Sasjs) {
-            if (res.result._webout.length < 1)
-              throw new JobExecutionError(
-                0,
-                'Job execution failed',
-                parsedSasjsServerLog
-              )
-          }
+          const resObj =
+            this.serverType === ServerType.Sasjs
+              ? {
+                  result: res.result._webout,
+                  log: parsedSasjsServerLog
+                }
+              : res
 
           this.requestClient!.appendRequest(resObj, sasJob, config.debug)
 
+          if (
+            this.serverType === ServerType.Sasjs &&
+            res.result._webout.length < 1
+          ) {
+            throw new JobExecutionError(
+              0,
+              'Job execution failed',
```

Comment:
Is there more information we can provide here?  http response code perhaps?  And perhaps we can be more specific (eg the job executed, but no webout was returned)

Reply:
suggestion:

> No webout was returned by job $(program). Servertype is SASJS and the calling function is WebJobExecutor. Please check the SAS log for more info.